app/app.go

```diff
@@ -3,20 +3,15 @@ package app
 import (
 	"context"
 	"encoding/json"
-	"io"
-	"net/http"
-	"os"
-	"path/filepath"
-	"strings"
-
-	"github.com/CosmWasm/wasmd/x/wasm"
-	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
-	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I don’t think this ought to be removed?

Reply:
(Specifically the Prometheus import)

Reply:
Yes we should keep Prometheus

Reply:
in this case Prometheus only used in wasm